flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/functions/SqlFunctionUtils.java

```diff
@@ -371,6 +371,11 @@ public static String repeat(String str, int repeat) {
         return EncodingUtils.repeat(str, repeat);
     }
 
+    /** Returns an expr where all characters in from have been replaced with those in to. */
+    public static String translate3(String str, String search, String replacement) {
+        return org.apache.commons.lang3.StringUtils.replaceChars(str, search, replacement);
```

Comment:
Could you please clarify what is the reason to use `StringUtils#replaceChars` instead of `replace` from this class?

Reply:
Replacechars () is a static method of the stringutils class, which is used to replace all search characters with replacement characters. And replace() is a static method of the stringutils class, which optionally accepts the first maximum number of matches of the search string to be replaced in a given text.
Example:
```java
public static void main(String[] args) {
    String s1 = StringUtils.replaceChars("AaaBabbCc", "ab", "123");
    String s2 = StringUtils.replace("AaaBabbCc", "ab", "123");
    System.out.println("replaceChars:" + s1);
    System.out.println("replace:" + s2);
}
```
> replaceChars:A11B122Cc
replace:AaaB123bCc

Reply:
I didn't say to use `StringUtils#replace` I asked about usage of `replace` from this class `SqlFunctionUtils#replace`

Reply:
StringUtils#replaceChars replaces a set of characters with a single character, while SqlFunctionUtils#replace replaces a substring with another string.
example：
``` java
String replaced = StringUtils.replaceChars("AaBbCc", "abc", "123");
System.out.println(replaced); // "A1B2C3"
String replaced = SqlFunctionUtils.replace("AaBbCc", "abc", "123");
System.out.println(replaced1); // "AaBbCc"
```
To meet the functionality requirements of Hive or Spark, it is necessary to use StringUtils#replaceChars. This is also the approach taken in Calcite.